commit 5593ef45d709382b4aeddc65660634af2fe1b0ba
Author: bertin <macportsuser@macports.org>
Date:   Fri Oct 10 19:16:21 2025 +0200

    /opt/local/linux-ports/audio/audacity/files/patch-cmake-dont-strip.diff

diff --git a/cmake-proxies/cmake-modules/AudacityFunctions.cmake b/cmake-proxies/cmake-modules/AudacityFunctions.cmake
index f9f618239..37297ffc3 100644
--- a/cmake-proxies/cmake-modules/AudacityFunctions.cmake
+++ b/cmake-proxies/cmake-modules/AudacityFunctions.cmake
@@ -583,7 +583,7 @@ function( audacity_module_fn NAME SOURCES IMPORT_TARGETS
       if (CMAKE_BUILD_TYPE MATCHES "Debug|RelWithDebInfo")
          set(nostrip 1)
       else()
-         set(nostrip 0)
+         set(nostrip 1)
       endif()
       add_custom_command(
          TARGET "${TARGET}"
